docs/content/en/docs/faq/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Frequently Asked Questions
+title: FAQ
```

Comment:
Why not keep the acronym expansion?

Reply:
It's quite long on the menu bar takes 2 lines, this look better a bit imho

Reply:
shortend also the contribution title, those are not that much of an issue, since we have others which multi lines, just a bit nicer